packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -38,9 +38,13 @@ const _removeGranuleFromCmr = async (granule, collectionId) => {
  * @param {Object} params
  * @param {Knex} params.knex - DB client
  * @param {Object} params.pgGranuleRecord - A Postgres granule record
- * @param {string} params.pgCollection - A Postgres Collection record
- * @param {Object} params.granulePgModel - Instance of granules model for PostgreSQL
- * @param {Object} params.granuleDynamoModel - Instance of granules model for DynamoDB
+ * @param {string} [params.pgCollection] - A Postgres Collection record
+ * @param {Object} [params.granulePgModel=new GranulePgModel()]
+ * - Instance of granules model for PostgreSQL
+ * @param {Object} [params.granuleDynamoModel=new models.Granule()]
```

Comment:
Probably not worth holding this PR up for, but we should have a more specific type for this.

Reply:
Sure it is.  3ab40c4345.        To some degree we need to have a conversation on the bounds on ts-check related concerns/jsdoc typings/etc for consistency in review.       For example, the hack we're doing for this object with `cmr_link`.